packages/fiori/src/ProductSwitch.js

```diff
@@ -0,0 +1,126 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import ItemNavigation from "@ui5/webcomponents-base/dist/delegate/ItemNavigation.js";
+import ResizeHandler from "@ui5/webcomponents-base/dist/delegate/ResizeHandler.js";
+import Integer from "@ui5/webcomponents-base/dist/types/Integer.js";
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import ProductSwitchTemplate from "./generated/templates/ProductSwitchTemplate.lit.js";
+
+// Styles
+import ProductSwitchCss from "./generated/themes/ProductSwitch.css.js";
+
+
+const metadata = {
+	tag: "ui5-product-switch",
+	properties: {
+		/**
+		 * Indicates how many columns are displayed.
+		 * @private
+		 */
+		desktopColumns: {
+			type: Integer,
+		},
+	},
+	slots: {
+		/**
+		 * Defines the items of the <code>ui5-product-switch</code>.
+		 *
+		 * @type {HTMLElement[]}
+		 * @slot
+		 * @public
+		 */
+		"default": {
+			propertyName: "items",
+			type: HTMLElement,
+		},
+	},
+};
+
+/**
+ * @class
+ * <h3 class="comment-api-title">Overview</h3>
+ *
+ * The <code>ui5-product-switch</code> is a Fiori specific web component that is used in <code>ui5-shellbar</code> and allows the user easy switching between products.
```

Comment:
The <code>ui5-product-switch</code> is a**n SAP** Fiori specific web component that is used in <code>ui5-shellbar</code> and allows the user **to easily switch** between products.